ci-operator/step-registry/dpu-operator/workflow/tests/dpu-operator-workflow-tests-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+token_dpu_operator_key=$(cat "/var/run/token/dpu-token/dpu-key")
+endpoint=$(cat "/var/run/token/dpu-token/url")
+
+job_url="https://${endpoint}/job/Lab144_DPU_Operator_Test/lastBuild"
+
+while : 
+do
+    job_check=$(curl -k -s --resolve "${endpoint}:443:10.0.180.88" "$job_url/api/json")
+    result=$(echo "$job_check" | jq -r '.result')
+    if [ $result != "null" ]; then
+	sleep $((RANDOM % 61 + 60))
```

Comment:
When the result is "OK" (assuming here), why do we need to sleep more than a minute here plus some random number of seconds?

Reply:
the reason is because if the result is okay, the other PR's script may need to finish running. I introduced randomness to this so that 2 PR's that are waiting for 1 to finish do not fire off at the same time.

Reply:
Wait can multiple PRs run this test at the same time?

Reply:
No they cannot and they should not. Essentially the tests "wait" to see if another job is failing and they will sleep while the other job finishes completion. After that once the job has freed up, it will fire off the request. So two PR's cannot run the jenkins job at the same time.

Reply:
But yes multiple PR's should be able to run this e2e test at the same time, but they should not trigger the same jenkins job at the same time